code/modules/mob/interactive.dm

```diff
@@ -139,7 +139,7 @@
 /client/proc/toggleSNPC(var/mob/A in SSnpc.botPool_l)
 	set name = "Toggle SNPC Proccessing Mode"

```

Comment:
Names like this are why all the button positions changed. Can't we think of anything shorter?